react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java

```diff
@@ -194,7 +194,9 @@ private void updateSelectionIfNeeded(ReactAztecText view, @Nullable ReadableMap
         if ( selection != null ) {
             int start = selection.getInt("start");
             int end = selection.getInt("end");
-            view.setSelection(start, end);
+            if (view.getText().length() > end && start >= 0) {
```

Comment:
I think we can add the equality to `end` in the condition since the caret can actually be placed right after the last char.

Reply:
Also, let's add a line comment about why we're doing this here, wdyt?

Reply:
both addressed in 7b8955aa